### ISSUE TITLE
Include field indicators

### ISSUE THREAD

**Issue by edsu** [sul-dlss/marctable#11]
Brooks Travis asked in code4lib Slack if field indicators were included in the table output. They aren't currently but it should be fairly easy to add to the Avram schema and to the CSV and Parquet outputs.

**Comment by elkym** [comment]
Any progress on this? Tabular data from MaRC seems to be incredibly difficult to manage, and the indicators are one of the biggest challenges.

**Comment by edsu** [comment]
Yes, MARC isn't really expressible very well as tabular data. But marctable is here for people who want to work with that complexity, mostly in data science contexts where there are tools for working with CSV and Parquet data.

It sounds like you could use the indicators in the output?

**Comment by elkym** [comment]
Yeah. I've been actually trying to use PyMarc to build my own tabular conversion. If I can get it functional enough, I'm happy to offer the code to you. I'm not yet experienced enough to modify your code and push it back via github's version control.
